frontend/README.md

```diff
@@ -25,5 +25,5 @@ REDIRECT_URI=<redirect_uri>
 To run the app, type:
 
 ```shell
-make run-app
+npm run dev
```

Comment:
What is the motivation for removing Make? I feel it is a good abstraction to ease development.

Reply:
I have already added Make file again, but it needs to be config for NextJS environment